Add CI workflow (#25)
cleanlab/examples: DEVELOPMENT.md

    3. add a markdown block right above the code cell that imports the dependencies that reads:
    
        > Please install the dependencies specified in this requirements.txt file before running the notebook.

        where the requirements.txt text hyperlinks to the `requirements.txt` file located in that specific example's folder (the file created in step 1). Be sure to use the absolute github link (instead of relative path) as Google Colab will have no access to other files. 

        > eg. for the [find_label_errors_iris](find_label_errors_iris/find_label_errors_iris.ipynb) notebook, we would hyperlink the following url:
        > 
        > ```
        > https://github.com/cleanlab/examples/blob/master/find_label_errors_iris/requirements.txt
        > ```

- If the notebook takes a long time to run or is hard to auto-execute, add its folder name to the `ignore_folders` list in [run_all_notebooks.py](run_all_notebooks.py).

- Add the notebook to the [Table of Contents](https://github.com/cleanlab/examples#table-of-contents)
 table in the README, ideally grouping the newly added example with any other related examples.

> - After a new notebook has been added and pushed to `master` branch, refrain from changing the notebook and folder names, as many links referencing the example notebook will be used throughout cleanlab documentation, blog posts and more.

Reviewer: ```suggestion
- After a new notebook has been added and pushed to `master` branch, AVOID changing the notebook and folder names, as this may  break existing links referencing the example notebook throughout cleanlab documentation, blog posts, and more.
```

- Ensure that the jupyter notebook cells are executed in order. Additionally clear any cell blocks that are too large (eg. model training code that specifies accuracy for each epoch), it is ok if these do not have an execution number after being cleared.

- The second cell of the notebook (right after the `<h1>` title block, and right before the text introduction of the notebook) should be a markdown block containing the text:
    ```
    {{ badge }}
    ``` 
    This will allow our CI workflow to create a badge that will link to a Google Colab version of the notebook.

> 

Reviewer 2: 🙌
